crates/ln-dlc-node/src/transaction.rs

```diff
@@ -0,0 +1,23 @@
+use bitcoin::Txid;
+use std::fmt;
+use std::fmt::Display;
+use std::fmt::Formatter;
+use time::OffsetDateTime;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct Transaction {
+    pub txid: Txid,
+    pub fee: u64,
+    pub created_at: OffsetDateTime,
+    pub updated_at: OffsetDateTime,
+}
```

Comment:
🙃 Here we have another opportunity to encapsulate when the fee is updated with methods etc. The idea would be to be able to remove `pub` from all the fields here (and in `Channel` too).